examples/cpp/main.cpp

```diff
@@ -36,6 +36,8 @@ int main() {
   printf("Reading ADC Channels: 1 = %i, 2 = %i, 3 = %i, 4 = %i\n",
          adc.channel[0], adc.channel[1], adc.channel[2], adc.channel[3]);
 
+  set_pwm_channel_value(pwm_Channel::Ch0, 10);
```

Comment:
what 10 stands for in pwm ?

Reply:
hmm, the _value_ should be the PWM's duty cycle, it would be clearer if the function was named _set_pwm_channel_duty_cycle_.

Reply:
Is it a % ? Is it ms ? If so, is there validation and an error message for the user ? 
I just saw that we have zero documentation for the API, the rest one.

Reply:
Yeah, this is weird. The duty cycle is usually a value **clipped** between 0 and 1.

@RaulTrombin could you describe what's the expected range and reasoning behind `10`?

Reply:
It controls the turn off on the channel, at 2047 it's a 50% pwm's dc,
The range is [0..4095].